nixos/modules/system/boot/systemd.nix

```diff
@@ -73,7 +73,7 @@ let
       "systemd-journald.service"
       "systemd-journal-flush.service"
       "systemd-journal-catalog-update.service"
-      "systemd-journald-audit.socket"
+      ] ++ (optional (!config.boot.isContainer) "systemd-journald-audit.socket") ++ [
```

Comment:
This seems to be really not related to this PR. Can you explain what this is about?

Reply:
It is part of those LXC fixes from https://github.com/NixOS/nixpkgs/issues/9735#issuecomment-500164017
PR's title does not include for brevity.

Reply:
It wouldn't have hurt to explain it in the commit message IMHO.